server/controllers/user.controller.js

```diff
@@ -1,67 +1,121 @@
 ///////////////////////////// Import dependencies ///////////////////////////////
-const jwt = require('jsonwebtoken');
-const User = require('../models/user.model');
-const Authenticator = require('../services/Authenticator');
-const OnRegisterTokenVerifier = require('../services/OnRegisterTokenVerifier');
-const NewUserCreator = require('../services/NewUserCreator');
+const jwt = require("jsonwebtoken");
+const User = require("../models/user.model");
+const Authenticator = require("../services/Authenticator");
+const OnRegisterTokenVerifier = require("../services/OnRegisterTokenVerifier");
+const NewUserCreator = require("../services/NewUserCreator");
```

Comment:
We don't want to include code formatting changes in PR. If you are using a code formatting tool then add this file to its .ignore